utils.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"io/ioutil"
+)
+
+// Startup script used to find/format/mount all local SSDs in GCE.
+// Each disk is mounted to /mnt/data<disknum> and chmoded to all users.
+const gceLocalSSDStartupScript = `
+#!/usr/bin/env bash
```

Comment:
I thought this had to be the first line of the file. This is making it the second.

Reply:
Still works, but yes, you're right. Done.